change instanceOf within canShoot method to getClass.

JordiBerns opened the pull request:
add bigEnemy and hp to small/big enemies.
can't use the hp yet.

JordiBerns committed:
change instanceOf within canShoot method to getClass.
add bigEnemy and hp to small/big enemies.
can't use the hp yet.

tomasheldeweg reviewed (approved):
Aah ik snapte eerst niet hoe die getClass() toegepast moest worden. 
Dit is volgens mij ook wat raymond als alternatief gaf voor instanceof dus laten we maar doen!

JordiBerns committed:
add abstract PiecesOnPlayingField class.

add abstract method determineIfCanShoot.

classes Character, Obstacle, Item extend PiecesOnPlayingField.

change the Object array to a PiecesOnPlayingField array.

add boolean if Object can damage or not, if not then loop will stop because of false.

add constructors to enemies and obstacles.
